src/Raven.Server/Documents/ETL/Providers/Raven/RavenEtlItem.cs

```diff
@@ -1,15 +1,28 @@
-﻿namespace Raven.Server.Documents.ETL.Providers.Raven
+﻿using Raven.Client.Documents.Operations.Counters;
+
+namespace Raven.Server.Documents.ETL.Providers.Raven
 {
     public class RavenEtlItem : ExtractedItem
     {
-        public RavenEtlItem(Document document, string collection) : base(document, collection)
+        public RavenEtlItem(Document document, string collection) : base(document, collection, EtlItemType.Document)
         {
            
         }
 
-        public RavenEtlItem(Tombstone tombstone, string collection) : base(tombstone, collection)
+        public RavenEtlItem(Tombstone tombstone, string collection, EtlItemType type) : base(tombstone, collection, type)
         {
            
         }
+
+        public RavenEtlItem(CounterDetail counter, string collection)
```

Comment:
I think that we need here another, RavenCounterEtlItem that inherits from RavenEtlItem.

Reply:
I can do that but not sure if it's really worth. The logic is based on `EtlItemType` anyway (the same concept as we have for `ReplicationBatchItem`). What advantages do you see of introducing a new type here?

Reply:
I think that because we are going to add other sub entities like time series etc, we might want to add some better seperation between records, other than one general entity for everything.

Reply:
I'll try to make `RavenEtlItem` an abstract class so we could have separate implementations like: `RavenDocumentEtlItem` `RavenCounterEtlItem` etc

Reply:
OK